File: ad-hoc/java/3475.java
import java.util.Scanner;

public class NumeroPorExtenso {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String numero = scanner.nextLine();

        switch (numero) {
            case "zero":
                System.out.println("0");
                break;
            case "um":
                System.out.println("1");
                break;
            case "dois":
                System.out.println("2");
                break;
            case "tres":
                System.out.println("3");
                break;
            case "quatro":
                System.out.println("4");
                break;
            case "cinco":
                System.out.println("5");
                break;
            case "seis":
                System.out.println("6");
                break;
            case "sete":
                System.out.println("7");
                break;
            case "oito":
                System.out.println("8");
                break;
            case "nove":
                System.out.println("9");
                break;
            case "0":
                System.out.println("zero");
                break;
            case "1":
                System.out.println("um");
                break;
            case "2":
                System.out.println("dois");
                break;
            case "3":
                System.out.println("tres");
                break;
            case "4":
                System.out.println("quatro");
                break;
            case "5":
                System.out.println("cinco");
                break;
            case "6":
                System.out.println("seis");
                break;
            case "7":
                System.out.println("sete");
                break;
            case "8":
                System.out.println("oito");
                break;
            case "9":
                System.out.println("nove");
                break;
            default:
                System.out.println("Número não reconhecido");
        }
    }
}

File: ad-hoc/javascript/3475.js
const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question((numero) => {
    switch (numero) {
        case 'zero':
            console.log('0');
            break;
        case 'um':
            console.log('1');
            break;
        case 'dois':
            console.log('2');
            break;
        case 'tres':
            console.log('3');
            break;
        case 'quatro':
            console.log('4');
            break;
        case 'cinco':
            console.log('5');
            break;
        case 'seis':
            console.log('6');
            break;
        case 'sete':
            console.log('7');
            break;
        case 'oito':
            console.log('8');
            break;
        case 'nove':
            console.log('9');
            break;
        case '0':
            console.log('zero');
            break;
        case '1':
            console.log('um');
            break;
        case '2':
            console.log('dois');
            break;
        case '3':
            console.log('tres');
            break;
        case '4':
            console.log('quatro');
            break;
        case '5':
            console.log('cinco');
            break;
        case '6':
            console.log('seis');
            break;
        case '7':
            console.log('sete');
            break;
        case '8':
            console.log('oito');
            break;
        case '9':
            console.log('nove');
            break;
        default:
            break;
    }
    rl.close();
});
